### PR TITLE
Fix react component version of export plan dashboard.

### PR DESCRIPTION
**Pull request by jakehopking**
### Workflow

 - [X] Ticket exists in Jira  https://uktrade.atlassian.net/browse/GP2-1192 
 - [X] Jira ticket has the correct status.
 - [X] [Changelog](CHANGELOG.md) entry added.
 
### Reviewing help

 - [X] Includes screenshot(s) - ideally before and after, but at least after

### Housekeeping

 - [X] Requirements have been re-compiled.

### Merging

 - [X] This PR can be merged by reviewers. (If unticked, please leave for the author to merge)

Screenshot 2020-12-07 at 17 33 43

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -39,6 +40,7 @@
 
 ### Fixed bugs
 
+- GP2-1192 - Landing page menu on right - react component updates.
```
Comment (webbyfox) [inline CHANGELOG.md:43]: This needs to go on top of files under line 7

Comment (webbyfox) [reply CHANGELOG.md:43]: also, CL looks outdated

Comment (jakehopking) [reply CHANGELOG.md:43]: why do we have two 'Fixed bugs' sections?

**Review by webbyfox — APPROVED** [review]
LGTM

**Pull request merged**